### ISSUE TITLE
Redo the content of the src/readme.html

### ISSUE THREAD

**Issue by warrior79** [ClassicPress/ClassicPress#33]
The content should be changed, and we should remove any links from the wordpress.org domain.
References to names of persons should be removed. e.g: Matt Mullenweg

**Comment by nylen** [comment]
I agree, the readme.html needs some updates, but I think we should give credit to those who came before us.  ClassicPress did not emerge out of nothing - what we have today is the result of many people putting years of hard work into WordPress.

**Comment by senlin** [comment]
@warrior79 when doing part 2 of #3 we decided to leave readme.html and license.txt as is for now. They indeed both need to receive a makeover and I agree with @nylen that we have to give credit where credit is due.

**Comment by nylen** [comment]
Let's get this done in time for our first beta release in roughly a month.

**Comment by nylen** [comment]
We are out of time for beta1, milestoning this for beta2.

**Comment by jeffmcneill** [comment]
I don't mind taking a stab at this, but there will be about 20 questions that need to be discussed/answered regarding changes (and they will need to have some final agreement). What is the best place for that discussion?

**Comment by nylen** [comment]
@jeffmcneill that would be great!

Here is fine, or in Slack `#core` if you want more real-time answers.

**Comment by senlin** [comment]
@jeffmcneill - apologies, I only just now noticed that you were planning to work on this too. Perhaps use the PR I just did as a base or use what you can.

**Comment by jeffmcneill** [comment]
@senlin No worries, busy here so your contribution is appreciated!
